### DISCUSSION TITLE
Am I banned from suggesting enhancements?

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#8272] [Q&A] [answered]
I created some (maybe five) enhancement suggestions. And you guys didn't respond or say anything about it, so I'm just thinking am I banned from suggesting enhancements, or it's something else?
![Screenshot_2021-09-26-22-40-19](https://user-images.githubusercontent.com/70109540/134821812-dc7f6348-71a0-4b28-bd31-02fbcca45b49.png)

**Comment by Frenzie** [comment]
I don't really have anything pertinent to say about statistics, and to a lesser extent the sync and the footer. I think having the ticket open there increases the chances that someone will pick up on it but our spare time is limited.

With regard to statistics, I'm not sure to what extent syncing such a thing could scale on the server side.

**Comment by poire-z** [comment, marked as answer]
> so I'm just thinking am I banned from suggesting enhancements, or it's something else?

Don't think that !
Personally, I had read all your issues when you made them - and I just had nothing much to say/add about them:
- I know nothing about sync (and not many people do, so these issues will probably don't get many answers)
- I don't care much about statistics
- and for "footer preview" and "hide the time at certain times", my first reaction was "come on!" - and thought I'd better shut up :)

So, let them sit, someone may come by and want it too, and implement it , or ask more questions and suggestions, and we may end up with something more generic/global...

(You did some other suggestions in the past that got a better destiny - so I assume KOReader has become perfect, if you now can only find suggestions are at such remote territories :)

**Comment by Zeyadas** [comment]
> my first reaction was "come on!"

That's happening to me, sometimes I find myself thinking weirdly. 

> so I assume KOReader has become perfect. 

I believe in that too, it's really perfect, I don't use any other app for reading, and that's why I (request) strange features. 
In my imagination I would remove the android operating system on my tablet and install koreader alone to make it (koreader tablet). 

> I don't care much about statistics. 

Me too sir, but I bought a new device (android tablet) and I thought if there is a way to (merge) the statistics from both devices so I just (requested) that. 
> and for "footer preview" and "hide the time at certain times", my first reaction was "come on!" - and thought I'd better shut up :) 

And I'd better shut up, I'll hide it manually when I'm reading in the bed. 

You guys are awesome, keep the good work and thank you all.
